src/BLI.jl

```diff
@@ -9,3 +13,53 @@ end
 function R2Calc(R::Real, Kdis::Real, tps)
     return R * exp.(-Kdis * tps)
 end
+
+
+function importData(cond)
+    df = CSV.read(cond)
+    conc = df[1,2:end]
+    tps = df[5:end,1]
+    measVal = df[5:end,2:end]
+    return conc, parse.(Float64, tps), measVal
+end
+
+function plotBLI(cond)
+    conc, tps, bindData = TAMode.importData(cond)
+    bindData = Matrix(bindData)
+    plot(parse.(tps), parse.(bindData), title=cond, 
+         label=[conc[1] conc[2] conc[3] conc[4] conc[5] conc[6] conc[7] conc[8]], lw=3)
+end
+
+@model BLI(cond) = begin
+    Tshift = 822.2
+    Kon ~ Normal(6., 0.5)
+    Kdis ~ Normal(2., 1.)
+    
+    conc, tps, bindData = importData(cond)
```

Comment:
@micah-bryant you should import the data outside of the `@model` object and then pass it in (like you do with `cond`). I think one of the things slowing everything down is every time the model is called it's reimporting the data.

Reply:
fixed in the recent commit